package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "unnamed-inventory-manager",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "test",
+    "authlocalinvoke": "sam local invoke IPAuthorizerLambda --event ip-authorizer/event.json --env-vars ip-authorizer/env.json",
+    "backlocalinvoke": "sam local invoke BackendLambda --event backend-lambda/event.json --env-vars backend-lambda/env.json",
+    "apilocalinvoke": "sam local start-api --env-vars env.json"
```

Comment:
This should use an env.json that's in the backend lambda folder

Reply:
What do you mean by this? Put all env variables in the backend lambda folder? Then rewrite the CLI commands to use that path?

Reply:
You're right, that was vague. The last apilocalinvoke command a) doesn't specify which SAM resource you're starting the api on (it's either the backend lambda itself or the api gateway that you'll configure in the next story and I'm not sure which) and b) the env.json file it's looking for is a sibling of the package json file when it should be one of the env.json files in one of the children.
So it should be something like:
"apilocalinvoke": "sam local start-api BackendLambda --env-vars backend-lambda/env.json"

Reply:
`We still want an individual env and event json for each function?`
Yes we do

Reply:
![image](https://github.com/Andrewdt97/unnamed-inventory-manager/assets/137709386/ddd79f0a-6a9e-484e-81ca-bd158675670f)

Reply:
Added individual env and event files back @Andrewdt97